client/state/sites/plans/actions.js

```diff
@@ -62,3 +63,21 @@ export function fetchSitePlansCompleted( siteId, plans ) {
 		plans: map( plans, createSitePlanObject )
 	};
 }
+
+/**
+ * Returns an action object to be used in updating an object containing
+ * the plans for the given site.
+ *
+ * @param  {Object} siteId ID of the concerned site
+ * @return {Object}        Action object
+ */
+export function updateSitePlans( siteId ) {
```

Comment:
What do you think about naming this something like `refreshSitePlans`? `updateSitePlans` sounds to me like we're trying to change them on the server.